internal/params/flags.go

```diff
@@ -33,6 +33,8 @@ const (
 	BranchFlag                   = "branch"
 	BranchFlagSh                 = "b"
 	ScanIDFlag                   = "scan-id"
+	CodeRepositoryFlag           = "code-repository-url"
+	CodeRepositoryFlagUsage      = "Code repository URL (optional for self-hosted Azure DevOps)"
```

Comment:
it's only for Azure? not for any pr deco?

Reply:
changed to 
Code repository URL (optional for self-hosted SCMs)